data/logs_model/logs_producer/splunk_logs_producer.py

```
@@ -62,6 +65,12 @@ def __init__(
             raise Exception(
```

if splunk is unable to connect here we don't want to re-raise this exception